configuration.adoc

```diff
@@ -1803,6 +1810,101 @@ Example, "userSMIMECertificate, UserCertificate"
 
 To set up another external LDAP server, click *Add Configuration*.
 
+=== Setting up server based S/MIME feature
+
+==== Prerequisites
+
+Same as the previous S/MIME version except that the Java is not required on client machine
+
+==== S/MIME License
+
+Same as the previous S/MIME version
+
+==== Enable S/MIME Feature
+
+Same as the previous S/MIME version
+
+==== Importing S/MIME Certificates
+
+Same as the previous S/MIME version except that the recipients' public-key certificate no longer need to be stored into Local OS or browser keystore.
+The certificate can be published to all other places mentioned in previous S/MIME version.
+
+==== List of LDAP attributes introduced in new S\MIME feature
+
+. zimbraSmimeOCSPEnabled
+
+* Used by server at the time of validating the user as well as public certificates
+* If TRUE, the revocation check will be performed during certificate validation
+* If FALSE, the revocation check will not be performed during certificate validation
+
+. zimbraSmimePublicCertificateExtensions
+
+* The supported public certificate file extensions separated by comma
+* Contains the list of supported formats for userCertificate LDAP attribute
+* Default values: cer,crt,der,spc,p7b,p7r,sst,sto,pem
+* Zimbra web client retrieves from server the supported file formats/extensions for public certificate upload
+
+. zimbraSmimeUserCertificateExtensions
+
+* The supported public certificate file extensions separated by comma.
+* Contains the list of supported formats for userSmimeCertificate LDAP attribute
+* Default values: p12,pfx
+* Zimbra web client retrieves from server the supported file formats/extensions for user certificate upload
+
+==== Process for Adding the CA certificate to the mailbox truststore for S/MIME
+
+S/MIME uses the mailbox trust store path and its password defined in localconfig.xml
+
+The key names are:
+
+* mailboxd_truststore
+* mailboxd_truststore_password
+
+If the key is not defined in localconfig.xml, by default the values of mailboxd_truststore and mailboxd_truststore_password are:
+
+* <zimbra_java_home>/jre/lib/security/cacerts
+* changeit
```

Comment:
Removed the pwd field value. Keeping only mailboxd_truststore value. Just to let the reader know the default truststore it is referring to.